### PR TITLE
feat: values page

### LINKED ISSUE #140: Implement Values Page
## Description

The values page design can be found [here](https://www.figma.com/file/msxmKajRPLYiDDmsP60Ndh/Design-System-v1.1?type=design&node-id=1411-63360&mode=design&t=e8I6pyP6fiWYMKk1-0)

## Tasks

- [ ] Implement the new page
- [ ] Add the link to the navigation

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #140

### CODE REVIEW

File: components/values/ValueContact.vue [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+const { t } = useI18n();
+
+const contacts = [
+  {
+    title: 'Email',
+    description: t('values.contact_section.email.description'),
+    href: 'mailto:info@rotki.com',
+    icon: 'chat-smile-2-line',
```
Comment (kelsos) [inline components/values/ValueContact.vue:9]: Ok so instead of copy pasting the urls/mail addresses do you think we can move them instead to config?

**Review by kelsos — CHANGES_REQUESTED** [review]
So there are a few minor comments but overall it looks good.
I checked both mobile/desktop layouts and the look great.

**Review by tewshi — APPROVED** [review]
lgtm!

**Review by kelsos — APPROVED** [review]
lgtm, thanks :)

**Pull request merged**